inc/views/pluggable/pagination.php

```diff
@@ -128,6 +128,115 @@ public function filter_localization( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Determine whether to show the Jump to page HTML input.
+	 * 
+	 * @return bool
+	 */
+	private function show_jump_to_page_input() {
+		return ( ! $this->has_infinite_scroll() && get_theme_mod( 'neve_enable_jump_to_pagination' ) ) ? true : false;
+	}
+
+	/**
+	 * Create jump to navigation inputs.
+	 * 
+	 * @return mixed $markup HTML to output on page.
+	 */
+	private function create_jump_to_html() {
+
+		// Value escaped once we confirm it's not empty, we never save this value to the DB.
+		$request = $_SERVER['REQUEST_URI'] ?? ''; //phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		
+		if ( empty( $request ) ) {
+			return;
+		}
+
+		$request = esc_attr( $request );
+
+		global $wp_query;
+
+		$search_query = esc_attr( get_search_query() );
+		$search_input = ! empty( $search_query ) ? "<input id='s' type='hidden' value='$search_query' name='s' />" : '';
+
+		$button_text = apply_filters( 'neve_pagination_jump_button_text', __( 'Go', 'neve' ) );
+		$button_text = esc_attr( $button_text );
+
+		$max_num_pages = absint( $wp_query->max_num_pages );
+
+		$current_page = ! empty( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : '';
+
+		$markup = <<<MARKUP
```

Comment:
Please use standard sintax, like `echo '<div id="nv-pagination-jump">';` We don't use heredoc syntax.

Reply:
@cristian-ungureanu I'm returning the markup here so thats why i didn't use echo, I updated the code to not use heredoc

Reply:
@UVLabs, you can collect all your code in a variable and then return it if you need it that way.